root/io/filemerger/execTestMultiMerge.C

```diff
@@ -67,26 +67,28 @@ int testSimpleFile(const char *filename, Long64_t entries, Int_t compSetting, Lo
    return 0;
 }
 
-
 int execTestMultiMerge()
 {
    Int_t result = 0;
-   result += testMergedFile("mfile1-4.root",1,4938, kIs32bits ? 3 : 0);
-   result += testMergedFile("mzfile1-4.root",206,4992, kIs32bits ? 2 : 0);
-
-   result += testSimpleFile("hsimple.root",25000,1,414415, kIs32bits ? 10 : 8);
-   result += testSimpleFile("hsimple9.root",25000,9,432029,3);
-   result += testSimpleFile("hsimple9x2.root",2*25000,9,851123,9);
-   result += testSimpleFile("hsimple209.root",25000,209,394077,8);
-   result += testSimpleFile("hsimpleK.root",5*25000,209,1917395,16);
+   int hsimpleFTolerance = 6; // 5 for non fst builds
+   //result += testMergedFile("mfile1-4.root",1,4938, kIs32bits ? 3 : 0);
+   result += testMergedFile("mzfile1-4.root",206,5819, kIs32bits ? 2 : 0);
+   result += testMergedFile("mlz4file1-4.root",406,5840, kIs32bits ? 2 : 0);
+   result += testMergedFile("mzlibfile1-4.root",106,5744, kIs32bits ? 2 : 0);
+   result += testSimpleFile("hsimple.root",25000,1,519464, kIs32bits ? 10 : 8);
+   result += testSimpleFile("hsimple9.root",25000,9,518137,3);
+   result += testSimpleFile("hsimple9x2.root",2*25000,9,1020462,9);
+   result += testSimpleFile("hsimple209.root",25000,209,395932,8);
```

Comment:
> or I can change from compression level from 1 to 101 and from 9 to 109, and etc. for hsimple, hsimple9 and hsimple9x2 files?

This is a good idea in itself, please **add** (not change) those explicitly .  **But**, since the default is not changing (yet), the size of those files should not be changing ... or am I missing something?

> I can add separate test macros root/io/filemerger/execTestMultiMergeLZ4.C just now

I am not sure how this would help .....